crates/chain/src/indexer/mod.rs

```diff
@@ -0,0 +1,31 @@
+use bitcoin::{OutPoint, Transaction, TxOut};
```

Comment:
Can this file be `src/indexer.rs` instead. It's easier to find it that way.

Reply:
Sorry @ValuedMammal I thought `indexer/mod.rs` was the more common rust style. It was a long time ago but apparently now `src/indexer.rs` is the recommended way so I'm good with reverting back to `src/indexer.rs`. 
https://doc.rust-lang.org/reference/items/modules.html#module-source-filenames